releasenotes/2.1.0.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.1.0
+date: 2018-02-21 12:00:00
+release_notes: true
+permalink: /releasenotes/2.1.0/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Compute Improvements](#computeimprovements)
+1. [BlobStore Improvements](#blobstoreimprovements)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.0 represents an important milestone for the project. We are proud to support all major cloud providers in the marketplace with a mature, stable, codebase that is ready for production. You can [read the official announcement here](https://s.apache.org/jclouds210). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.1.0 features include:
+
+* Modernize Openstack support
```

Comment:
here I'd say "up to OpenStack Pike", which is the latest OpenStack version and it work fine with 2.1.0.

Reply:
done